src/patternfly/components/JumpLinks/jump-links.scss

```diff
@@ -98,6 +101,18 @@
     border: solid var(--pf-c-jump-links__list--before--BorderColor);
     border-width: var(--pf-c-jump-links__list--before--BorderTopWidth) var(--pf-c-jump-links__list--before--BorderRightWidth) var(--pf-c-jump-links__list--before--BorderBottomWidth) var(--pf-c-jump-links__list--before--BorderLeftWidth);
   }
+
+  .pf-c-jump-links__list {
+    --pf-c-jump-links__list--PaddingTop: 0;
+    --pf-c-jump-links__list--PaddingBottom: 0;
+    --pf-c-jump-links__link--PaddingTop: var(--pf-c-jump-links__list--list__link--PaddingTop);
```

Comment:
I thought we were only using `--` in `list--list` for a nested item that is a component. eg, `list--c-button`. This conflicts with the syntax we use for adjacent items, for styling `__list + __list` - the var syntax for that would be `list--list`. The same applies if we have an element adjacent to a component, too, eg `__list + c-button`, but I think we compromised on that for readability if I recall. Personally I think we should use `__` everywhere for children, whether it's an element of a component or separate component, so we leave room to style adjacent elements with `--` between the names in a var. wdyt?

Reply:
Ah yes sorry I always get confused with this syntax, agreed changed it to `__list`

Reply:
I think it's confusing, too, because we do it different ways. I'll add an agenda item for our next CSS meeting and we can go over whether it still makes sense to use `--c-component` for nested items, or if we can be consistent and use `__`.